console/package.json

```diff
@@ -5,10 +5,11 @@
 	"author": "Dave Freeman <dave.freeman@lge.com>",
 	"main": "src/index.js",
 	"scripts": {
-		"serve": "enact serve",
-		"pack": "enact pack",
-		"pack-p": "enact pack -p",
-		"watch": "enact pack --watch",
+		"config": "if [ ! -e config.js ]; then cp config.sample.js config.js ; fi",
```

Comment:
if-statement won't work cross-platform. May want to consider moving this logic into a js file, then  running in the package.json. For example, `"config":"node ./setup-config.js`.

Alternatively could add [cpy-cli]https://www.npmjs.com/package/cpy-cli) as a devDependency. Then you could update the config run task to: `"config":"cpy --no-overwrite --rename=config.js config.sample.js ."` which would be more simple/clean but add a devDependency.

Reply:
Incorporated the latter suggestion. Thanks!
_(If we need to support more complexity in the future, we can go with the former, rather than the latter.)_